test/unit/lib/asset_manager_access_limitation_test.rb

```diff
@@ -3,8 +3,8 @@
 class AssetManagerAccessLimitationTest < ActiveSupport::TestCase
   test "delegates to PublishingApi::PayloadBuilder::AccessLimitation to ask for users that can access item" do
     edition = FactoryBot.build(:edition)
-    access_limitation = { access_limited: { users: %w[uid-1] } }
+    access_limitation = { access_limited: { organisations: [1] } }
```

Comment:
It might be worth tweaking this stub just to make it more 'realistic' – since we expect the Content ID of the organisation (which is a String), not its ID (an Integer).

```suggestion
    access_limitation = { access_limited: { organisations: %w[org-1] } }
```